pub_sub/javascript/checkout/index.js

```diff
@@ -0,0 +1,20 @@
+import { DaprClient } from 'dapr-client';
+import { v4 as uuidv4 } from 'uuid';
+
+const SIDECAR_HOST = process.env.SIDECAR_HOST || "127.0.0.1";
```

Comment:
Can we change the variable name to DAPR_HOST

Reply:
Agreed, that would be consistent with the other examples